codecov.yml

```diff
@@ -1,3 +1,10 @@
+coverage:
+  status:
+    project:
```

Comment:
Do we want to set patch threshold too? https://docs.codecov.com/docs/commit-status#patch-status

Reply:
Hm. Probably.